assets/static/js/navbar.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function () {
+  const height = $(window).height();
+
+  $(".scrollable-menu").css("max-height", height);
+
+  // Style menu responsive
+  $(".navbar-toggler").on("click", function () {
```

Comment:
so bootstrap navbar is not responsible by default so that's the reason this snipped is needed, right ?

Reply:
`$(".scrollable-menu").css("max-height", height);` - Display a full size menu when there are a long list of links. This is a customization because the original implementation hides the last items

`$(".navbar-toggler").on("click", function () {` - We need to add some custom classes for menu when is displayed. For example, the color in the menu button is changed because the background pass from dark to light